### PR TITLE
RGRIDT-1066: Creating column sort api method

### PR DESCRIPTION
**Pull request by glundgren93**
This PR adds the required methods for column sorting.

### What was done
* Created API method for Column sorting and necessary methods for it to work.

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)

### CODE REVIEW

**Review by tmbp95 — APPROVED** [review]

File: code/src/OSFramework/Feature/IColumnSort.ts [hunk 1]
```diff
@@ -6,5 +6,9 @@ namespace OSFramework.Feature {
             IView {
         isGridSorted: boolean;
         clear(): void;
+        sortColumn(
+            columnID: string,
+            ascending: OSFramework.OSStructure.Sorting
```
Comment (JoaoFerreira-FrontEnd) [inline code/src/OSFramework/Feature/IColumnSort.ts:11]: you can delete "OSFramework."

File: code/src/GridAPI/Sort.ts [hunk 2]
```diff
@@ -25,4 +25,24 @@ namespace GridAPI.Sort {
...
     }
+
+    export function ColumnSort(
+        gridID: string,
+        columnID: string,
+        sorting: OSFramework.OSStructure.Sorting
+    ): void {
+        PerformanceAPI.SetMark('Sort.ColumnSort');
+
+        if (!OSFramework.Helper.IsGridReady(gridID)) return;
```
Comment (JoaoFerreira-FrontEnd) [inline code/src/GridAPI/Sort.ts:36]: why is not a true condition? and then encapsulate the following code...

**Review by JoaoFerreira-FrontEnd — CHANGES_REQUESTED** [review]

File: code/src/GridAPI/Sort.ts (hunk 2 above)
Comment (glundgren93) [reply code/src/GridAPI/Sort.ts:36]: We've been using this for all API methods. I just followed the pattern.

Comment (glundgren93) [reply code/src/GridAPI/Sort.ts]: Changed it

**Review by tmbp95 — APPROVED** [review]

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Review by tmbp95 — APPROVED** [review]

**Pull request merged**